src/utils/constants.ts

```diff
@@ -175,3 +175,26 @@ export const DEFAULT_PDNS_REGISTRY_STATE: PDNSContractJSON = {
 export const WARP_CONTRACT_BASE_URL = 'https://sonar.warp.cc/#/app/contract/';
 export const WARP_INTERACTION_BASE_URL =
   'https://sonar.warp.cc/#/app/interaction/';
+
+export const SMARTWEAVE_INTERACTION_TAGS = [
+  {
+    name: 'App-Name',
+    value: 'SmartWeaveAction',
+  },
+  {
+    name: 'Contract',
+    value: '',
+  },
+  {
+    name: 'Input',
+    value: '',
+  },
+];
```

Comment:
nit - make this a function that accepts `input` and `contract` as values and returns the array and use it above instead of `tags[1].value = contractTxId`

Reply:
https://github.com/ar-io/pdns-react/pull/241/commits/62cf09030fe95ab9c636d80160c0353354013077